data-pipeline-flink/dp-core/src/main/scala/org/sunbird/dp/core/job/BaseJobConfig.scala

```diff
@@ -59,7 +60,9 @@ class BaseJobConfig(val config: Config, val jobName: String) extends Serializabl
     properties.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaProducerBrokerServers)
     properties.put(ProducerConfig.LINGER_MS_CONFIG, new Integer(kafkaProducerLingerMs))
     properties.put(ProducerConfig.BATCH_SIZE_CONFIG, new Integer(kafkaProducerBatchSize))
-    properties.put(ProducerConfig.COMPRESSION_TYPE_CONFIG, "snappy")
+    if (kafkaProducerCompression != null && kafkaProducerCompression.nonEmpty) {
```

Comment:
With a default value, we need not do a check on whether kafkaProducerCompression is set.

Reply:
Got it, will remove the if statements

Reply:
if statements removed